psyneulink/library/compositions/compositionrunner.py

```diff
@@ -201,14 +201,11 @@ def run_learning(self,
             self._composition.run(inputs=minibatched_input, skip_initialization=skip_initialization, context=context, skip_analyze_graph=True, bin_execute=bin_execute, *args, **kwargs)
             skip_initialization = True
 
-        # FIXME: compiled run values differ from pytorch run
-        results = self._composition.parameters.results.get(context)[-1 * num_trials * minibatch_size:] # return results from last epoch
-        # if bin_execute is not False and bin_execute != 'Python':
-        #     results = [x for x in self._composition.parameters.results.get(context)[-1 * num_trials * minibatch_size:]] # return results from last epoch
-        # else:
-        #     results = [x[0] for x in self._composition.parameters.results.get(context)[-1 * num_trials * minibatch_size:]] # return results from last epoch
+        num_epoch_results = num_trials // minibatch_size # number of results expected from final epoch
```

Comment:
is `minibatch_size` guaranteed to be < `num_trials`? or is `num_epoch_results == 0` a legal outcome here?

Reply:
`minibatch_size` should always be `<=` `num_trials`.

Reply:
ok, I presume there's a check somewhere earlier enforcing it.

Reply:
added a check